test/Serilog.Sinks.OpenTelemetry.Tests/LogRecordBuilderTests.cs

```diff
@@ -180,4 +180,12 @@ public void IncludeTraceIdAndSpanId()
         Assert.Equal(logRecord.TraceId, PrimitiveConversions.ToOpenTelemetryTraceId(Activity.Current.TraceId.ToHexString()));
         Assert.Equal(logRecord.SpanId, PrimitiveConversions.ToOpenTelemetrySpanId(Activity.Current.SpanId.ToHexString()));
     }
+
+    [Fact]
+    public void IncludeTemplateBody()
+    {
+        var logRecord = LogRecordBuilder.ToLogRecord(Some.SerilogEvent(), null, IncludedData.TemplateBody, new());
+        Assert.NotNull(logRecord.Body);
+        Assert.Equal(Some.TestMessageTemplate, logRecord.Body.StringValue);
+    }
```

Comment:
Perhaps for the test name, `TemplateBodyIncludesMessageTemplateInBody()`? (A bit painfully verbose but spells out the goal.)

For the `Some.SerilogEvent()`, the test could end up inconclusive if the sample event has a template with no holes (so the test wouldn't be able to tell the difference between the template and the rendered message). Does the `SerilogEvent()` method allow specifying the template, e.g.:

```
Some.SerilogEvent("Hello, {Name}", "world")
```

or could we find a way to ensure this?

Reply:
Do you mean to be explicit about the template? `Some.SerilogEvent()` defines a default template of `"Message template {Variable}"`. If the default was changed to something like `Message template` then this test would not fail if the underlying implementation "broke". Is that what you are getting at?

In my opinion `Some.SerilogEvent()` is a bit odd because it lets you override the template but not the properties. I am thinking the best way to handle this might be to refactor the tests a bit. Something like

```diff
- internal static LogEvent SerilogEvent(DateTimeOffset? timestamp = null, Exception? ex = null, string messageTemplate = TestMessageTemplate)
+ internal static LogEvent DefaultSerilogEvent()
+ internal static LogEvent SerilogEvent(DateTimeOffset? timestamp = null, Exception? ex = null, string messageTemplate, IEnumerable<LogEventProperty> properties)
```

Thoughts?

Reply:
That looks good to me :+1:

(Sorry about the slow reply, spent a few nights away over the weekend, just catching up now 😅 )

Reply:
Made some updates, let me know what you think.